install_config/topics/creating_service_accounts.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// <install_config/registry/index#auth-enabled-registry.adoc>
+
+// Base the file name and the ID on the module title. For example:
+// * file name: doing-procedure-a.adoc
+// * ID: [id='doing-procedure-a']
+// * Title: = Doing procedure A
+
+[id='creating-service-accounts-authentication-tokens_{configuring_red_hat_registry}']
+= Creating Service Accounts and Authentication Tokens
```

Comment:
we need to be really really careful to disambiguate between "service accounts/tokens for the redhat registry" and "openshift cluster service accounts/tokens".

they are entirely unrelated.